src/engine/guidance/collapse_scenario_detection.cpp

```diff
@@ -123,7 +123,7 @@ bool isStaggeredIntersection(const RouteStepIterator step_prior_to_intersection,
 
     const auto angle = [](const RouteStep &step) {
         const auto &intersection = step.intersections.front();
-        const auto entry_bearing = intersection.bearings[intersection.in];
+        const auto entry_bearing = util::bearing::reverse(intersection.bearings[intersection.in]);
```

Comment:
@MoKob @vng what are next actions here - can we ticket reviewing the post-processing bearing calculations to work on as a next step and move forward here?